x-pack/plugins/apm/public/components/shared/PropertiesTable/__test__/NestedKeyValueTable.test.tsx

```diff
@@ -80,11 +81,13 @@ describe('FormattedValue component', () => {
   });
 
   it('should render null', () => {
-    expect(mount(<FormattedValue value={null} />)).toMatchSnapshot();
+    expect(mountWithIntl(<FormattedValue value={null} />)).toMatchSnapshot();
```

Comment:
Can you use `shallow` (instead of `mount`) and avoid brining i18n into the tests?